ee/clickhouse/views/test/test_clickhouse_funnel_person.py

```diff
@@ -53,6 +54,7 @@ def test_basic_format(self):
         self.assertTrue("id" in j["results"][0] and "name" in j["results"][0] and "distinct_ids" in j["results"][0])
 
     def test_basic_pagination(self):
+        cache.clear()
```

Comment:
Didn't set this in setUp, because I want things to be cached around so we know it's working fine generally. (it wasn't, so I fixed it a bit xD )